src/RTT_Format_Reader/RTT_Mesh_Reader.cc

```diff
@@ -48,8 +48,10 @@ void RTT_Mesh_Reader::transform2CGNS(void) {
       cell_def = Element_Definition::QUAD_4;
     else if (cell_name == "quad5")
       cell_def = Element_Definition::QUAD_5;
-    else if (cell_name == "quad6")
+    else if (cell_name == "quad6" || cell_name == "quad6a")
       cell_def = Element_Definition::QUAD_6;
+    else if (cell_name == "quad6o")
+      cell_def = Element_Definition::QUAD_6o;
```

Comment:
Sorry - I'm a little slow at getting back to you and following all of the changes.